languages/i18n/vp.json

```
@@ -256,19 +256,19 @@
 	"youhavenewmessagesmanyusers": "You have $1 from many users ($2).",
 	"newmessageslinkplural": "{{PLURAL:$1|a new message|999=new messages}}",
 	"newmessagesdifflinkplural": "last {{PLURAL:$1|change|999=changes}}",
-	"youhavenewmessagesmulti": "You have new messages on $1",
+	"youhavenewmessagesmulti": "Du har nyo zeposzta na $1",
 	"newtalkseparator": ",&#32;",
 	"editsection": "reforma",
 	"editold": "reforma",
 	"viewsourceold": "view source",
 	"editlink": "reforma",
 	"viewsourcelink": "view source",
-	"editsectionhint": "Edit section: $1",
-	"toc": "Contents",
+	"editsectionhint": "Kawari tel: $1",
+	"toc": "Sztof",

```

```suggestion
	"toc": "Teltumam",
```
Na andr tel, « Teltumam » brukena per toc. (ine `languages/messages/MessagesVp.php`)